docs/installLiberty.md

```diff
@@ -64,3 +64,58 @@ In certain cases, the Liberty license code may need to be provided in order to i
         }
     }
   ```
+
+### Using Maven artifact
+
+Use the `assemblyArtifact` to specify the name of the Maven artifact that contains a custom Liberty server or use one of the provided on the [Maven Central repository](http://search.maven.org/). 
+
+The Maven Central repository includes the following Liberty runtime artifacts:
+
+|Artifact ID | Versions | Description |
+| --- | ----------------- | ----------- |
+| [wlp-javaee7](https://repo1.maven.org/maven2/com/ibm/websphere/appserver/runtime/wlp-javaee7/) | 17.0.0.2, 17.0.0.1, 16.0.0.4, 16.0.0.3, 16.0.0.2, 8.5.5.9, 8.5.5.8, 8.5.5.7, 8.5.5.6 | Liberty runtime with all Java EE 7 Full Platform features. |
+| [wlp-webProfile7](https://repo1.maven.org/maven2/com/ibm/websphere/appserver/runtime/wlp-webProfile7/) | 17.0.0.2, 17.0.0.1, 16.0.0.4, 16.0.0.3, 16.0.0.2, 8.5.5.9, 8.5.5.8, 8.5.5.7, 8.5.5.6 | Liberty runtime with Java EE 7 Web Profile features. |
+| [wlp-kernel](https://repo1.maven.org/maven2/com/ibm/websphere/appserver/runtime/wlp-kernel/) | 17.0.0.2, 17.0.0.1, 16.0.0.4, 16.0.0.3, 16.0.0.2, 8.5.5.9, 8.5.5.8 | Liberty runtime kernel. |
+| [wlp-osgi](https://repo1.maven.org/maven2/com/ibm/websphere/appserver/runtime/wlp-osgi/) | 17.0.0.2, 17.0.0.1, 16.0.0.4, 16.0.0.3, 16.0.0.2, 8.5.5.9, 8.5.5.8 | Liberty runtime with features that support OSGi applications. |
+| [wlp-microProfile1](https://repo1.maven.org/maven2/com/ibm/websphere/appserver/runtime/wlp-microProfile1/) | 17.0.0.2, 17.0.0.1, 16.0.0.4, 16.0.0.3 | Liberty with features for a MicroProfile runtime. |
+
+
+Note: The group ID for these artifacts is: `com.ibm.websphere.appserver.runtime`.
+
+#### assemblyArtifact Properties
+
+These properties are used if `runtimeUrl` is not set.
+
+| Attribute | Description | Required |
+| --------- | ------------ | ----------|
+| groupId | Set Maven groupId to `com.ibm.websphere.appserver.runtime` to  download Liberty runtime archive from the Maven repository. If not set, the [Liberty repository](https://developer.ibm.com/wasdev/downloads/) will be used by default. | No |
```

Comment:
The more I think about this, I am not sure it should be removed.  It depends on what you discover related to using gradle dependency management to download the artifact.